armi/settings/fwSettings/globalSettings.py

```diff
@@ -29,6 +29,7 @@
 from armi import context
 from armi.settings import setting
 from armi.utils.mathematics import isMonotonic
+from . import tightCouplingSettings
```

Comment:
Pretty sure we prefer explicit imports.

Reply:
Yeah it wasn't working! Not really sure why...